### PR TITLE
Updated private link instructions

### PR DESCRIPTION
**Pull request by MarkWoulfeNeo4j**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates.

You can view the updated docs at https://neo4j-docs-aura-286.surge.sh

**Comment by MarkWoulfeNeo4j** [comment]
[Console Private Link](https://trello.com/c/opsj0H2X/252-console-private-link)

File: modules/ROOT/pages/platform/security.adoc [hunk 1]
```diff
@@ -112,14 +115,17 @@ To continue accessing Browser and Bloom, you can configure a https://cloud.googl
...
-* You use the private URL provided by support when you completed setting up the private endpoint. It will be different from the URL you used before.
+* You use the *Private URI* shown on the instance tile and in the instance details. It will be different from the *Connection URI* you used before.
 ====
 
 image::privateserviceconnect_03_browser_bloom_over_vpn.png[title="Accessing Browser and Bloom over a VPN"]
 
 ==== Enabling private endpoints
 
-To enable private endpoints using GCP Private Service Connect, please https://aura.support.neo4j.com/hc/en-us/requests/new[raise a support ticket], and we'll be in touch.
+To enable private endpoints using AWS PrivateLink:
```
Comment (MarkWoulfeNeo4j) [inline modules/ROOT/pages/platform/security.adoc:125]:
```suggestion
To enable private endpoints using GCP Private Service Connect: 
```

File: modules/ROOT/pages/platform/security.adoc [hunk 2]
```diff
@@ -163,14 +169,17 @@ To continue accessing Browser and Bloom, you can configure a VPN (Virtual Privat
...
-* You use the private URL provided by support when you completed setting up the private endpoint. It will be different from the URL you used before.
+* You use the *Private URI* shown on the instance tile and in the instance details. It will be different from the *Connection URI* you used before.
 ====
 
 image::azure_privatelink_03_browser_bloom_over_vpn.png[title="Accessing Browser and Bloom over a VPN"]
 
 ==== Enabling private endpoints
 
-To enable private endpoints using Azure Private Link, please https://aura.support.neo4j.com/hc/en-us/requests/new[raise a support ticket].
+To enable private endpoints using AWS PrivateLink:
```
Comment (MarkWoulfeNeo4j) [inline modules/ROOT/pages/platform/security.adoc:179]:
```suggestion
To enable private endpoints using Azure Private Link: 
```

**Review by MarkWoulfeNeo4j — COMMENTED** [review]
copy paste fixes

**Review by nvitucci — APPROVED** [review]

**Pull request merged**